lib/index.js

```diff
@@ -62,17 +77,19 @@ AppsFlyer.prototype.track = function(track, fn) {
     appsflyer_id: options.appsFlyerId,
     eventName: track.event(),
     eventCurrency: track.currency(), // defaults to 'USD'
-    ip: track.ip(),
     eventTime: timestamp,
     af_events_api: 'true'
   };
+  
+  // Add ip address if event is client side
+  if (track.ip()) payload.ip = track.ip();
```

Comment:
If the event comes from a mobile library, it will be undefined. My thinking here is that it's better it is just not part of the payload than undefined.